R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -239,6 +446,10 @@ dim.TableHandle <- function(x) {
   return(x$dim())
 }
 
+#' @description
```

Comment:
All of the documentation should largely be ripped off from Python where it has gone through extensive review as to what it should say.  `Merge one or more TableHandles` is very unclear/imprecise compared to `Merges several tables into one table on the server.`
https://deephaven.io/core/client-api/python/code/pydeephaven.html#pydeephaven.Session.merge_tables